packages/docusaurus-types/src/index.d.ts

```diff
@@ -140,3 +140,10 @@ export interface ConfigureWebpackUtils {
   getCacheLoader: (isServer: boolean, cacheOptions?: {}) => Loader | null;
   getBabelLoader: (isServer: boolean, babelOptions?: {}) => Loader;
 }
+
+declare global {
```

Comment:
`@docusaurus/types` is only for `@docusaurus/core` typing. I think this needs to be done in the plugin analytic itself

Reply:
Nice, advise! I fixed it.